elasticsearch-model/spec/elasticsearch/model/module_spec.rb

```diff
@@ -47,7 +47,7 @@ def self.search(query, options={})
         end
       end
 
-      DummyIncludingModel.__send__ :include, Elasticsearch::Model
+      DummyIncludingModelWithSearchMethodDefined.__send__ :include, Elasticsearch::Model
```

Comment:
I think you need both lines, no?

Reply:
do I? wow good catch

Reply:
haha yeah I think you do, so you can test both scenarios. I was reading the code yesterday :)

Reply:
changed